robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -319,6 +320,18 @@ public static final class EXTENSION_ARM {
     public static final double TRAP_SCORE_ROTATIONS = 0; // TODO: TUNE
   }
 
+  public static final class LEDS {
+    public static final int PORT_NUMBER = 0;
+    public static final int STRIP_LENGTH = 30;
+
+    public static final Color WHITE = new Color(255, 255, 255);
```

Comment:
Don't need to create constants for your colors. WPILib provides constants for colors already. Do `Color.k` and intellisense should show you the list of available colors. Otherwise you can look at the `Color` class and find where the constants are in there to see the options.